rfcs/20201023-sig-models.md

```diff
@@ -0,0 +1,91 @@
+| **Sponsor**   | Thea Lamkin (thealamkin@google.com)                  |
+| **Updated**   | 2020-10-23                                           |
+
+## What is this group for?
+
+This group is for discussions and collaborations on enabling community
+contributions to [TensorFlow Model Garden](https://github.com/tensorflow/models)
+and [Tensorflow Hub](https://github.com/tensorflow/hub).
+
+SIG Models will focus on empowering the community to contribute state-of-the-art
+model implementation in TensorFlow 2. It will benefit the whole community by
+providing recommended implementations and models with reproducible results.
+
+SIG Models will have several subgroups (e.g., SIG Models Vision and SIG Models
+NLP) covering different machine learning areas. There are several SIG leads for
+each group to coordinate the contributions, run community events like contests,
+and maintain the code quality through the review process. Each group has the
+flexibility to operate differently.
+
+SIG is also a place for community discussions and sharing best practices of
+using TensorFlow 2 for state-of-the-art research. Furthermore, SIG provides
+product feedback to help TensorFlow to be improved.
+
+## Who will be part of it?
+
+SIG Models are open, membership will be entirely public, and anybody interested
+in model contributions or participating in the discussion can join the SIG.
+There can be different ways to participate:
+
+* Everyone in the community can pick GitHub model tasks and contribute models.
+* They can also join the regular SIG meetings or email groups to participate in
+  the discussions. To participate, request an invitation to join the SIG mailing
+  list. Archives of the mailing list are publicly accessible.
+
+We will identify SIG leads from the community to run the SIG Models groups.
+Initially, we plan to set up the SIG Models Vision group:
+
+* SIG Models Vision leads
+  * George Thiruvathukal, gkt@cs.luc.edu (Loyola University Chicago)
+  * Yung-Hsiang Lu, yunglu@purdue.edu (Purdue University)
+* Co-leads and sponsors from TensorFlow
+  * Jaeyoun Kim, jaeyounkim@google.com (TensorFlow Model Garden)
+  * Jing Li, jingli@google.com (TensorFlow Model Garden)
+  * Mike Liang, mliang@google.com (TensorFlow Hub)
+  * Shuangfeng Li, shuangfeng@google.com (TensorFlow Lite)
+
+## What initial problems will the group tackle?
+
+We will start with the SIG Models Vision group. Initially, the group leads will
+run a contest to motivate the community to contribute state-of-the-art computer
+vision models using TensorFlow 2. 
+
+For example:
+
+* Define/release model implementation tasks, rules, and evaluation criteria for
+  the contest
+* Review initial submissions and provide more support for selected high-quality
+  submissions
+* Review final submissions and announce results
+
+## What modes of communication do you intend to use?
```

Comment:
I think would be helpful to also define what is expected use-case.

Do we see it as python module, that can be imported downstream, hence models are tested and maintained.
Or just a collection of scripts, that contributed by different people at different times?

Reply:
This comment is connected to the models's reusable components topic.
Some times ago we had a ticket about this at https://github.com/tensorflow/community/issues/223

Reply:
There are many dups or semi-dups like https://github.com/tensorflow/addons/issues/1903#issuecomment-715897109 that confuse users and third party SIG maintainers.

Reply:
@lc0 @bhank Thank you for your suggestions and comments. We will add that as one of the first topics for open discussion after the SIG Models created. We want to start with a small scope and expand it later based on community inputs.